Functions.Templates/Templates/SqlInputBinding-Python/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "defaultFunctionName": "SqlInputBindingPython",
+    "description": "$SqlInputBinding_description",
```

Comment:
You need to make new ones - these currently refer to C#. Add to https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Resources/Resources.resx